scripts/provision-tenant.sh

```
@@ -49,9 +49,26 @@ export CDK_PARAM_TENANT_ID=$tenantId
+# admin-console UI の Application Console column が「Open ↗ (pooled)」 と表示される
+# bug が観測された (2026-05-18 testsilo tenant)。
+export TIER=$(echo "$tier" | tr '[:lower:]' '[:upper:]')
 export TENANT_ADMIN_EMAIL=$email
 
+# Issue #1029 follow-up: admin-console-hosting stack の CompetitorBootstrapTemplateUrl
+# output (= public S3 URL) を CodeBuild 側で読み直して cdk deploy の synth に流す。
+# pooled stack の runtime-config.json に bootstrap URL が空のまま焼かれると、
+# 競技者の Launch Stack deeplink で CFn が「TemplateURL must be a supported URL」 で
+# reject する (= raw.githubusercontent.com fallback では deploy 不可)。 stack 不在は
+# 空文字 fallback (= 初回 install と同じく frontend が GitHub raw fallback に倒れる)。
+export CDK_PARAM_COMPETITOR_BOOTSTRAP_TEMPLATE_URL=$(aws cloudformation describe-stacks \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)